sdk/portal/src/examples/monitoring-alerting.ts

```diff
@@ -0,0 +1,206 @@
+import type { FormattedExecutionResult } from "graphql";
```

Comment:
In the other examples I added a comment block on top to describe what this example is doing. As it is is ending up in the docs I would do this also here.

Reply:
see https://console-release.settlemint.com/documentation/building-with-settlemint/building-with-sdk/portal#deploy-contract